src/components/inspectors/LoopCharacteristics.vue

```diff
@@ -190,6 +185,9 @@ export default {
         }
       },
     },
+    loopType(value) {
```

Comment:
Please do not use watchers. Is it possible to change it to @change event?

Reply:
changed